src/snowflake/snowpark/session.py

```diff
@@ -635,6 +644,10 @@ def auto_clean_up_temp_table_enabled(self) -> bool:
         """
         return self._auto_clean_up_temp_table_enabled
 
+    @property
+    def query_compilation_stage_enabled(self) -> bool:
```

Comment:
just another thought, we might not need to expose this parameter to users? We can only have `self._query_compilation_stage_enabled`

Reply:
yeah. good point

Reply:
as discussed offline, we removed the client side user facing parameter control for now